### PR TITLE
[11.0][MIG]stock_orderpoint_automatic_creation

### PR DESCRIPTION
**Pull request by ValentinVinagre**
Finished the PR https://github.com/OCA/stock-logistics-warehouse/pull/429 from @andreasdsp.

### CODE REVIEW

**Comment by rousseldenis** [comment]
@ValentinVinagre so, this supersedes #429 ?

**Comment by ValentinVinagre** [comment]
yes

**Comment by rousseldenis** [comment]
Please preserve commit history from previous versions

**Comment by ValentinVinagre** [comment]
I don't know how do that. Any help?

**Comment by rousseldenis** [comment]
@ValentinVinagre You can read https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

And especially the part with the 'git format patch' part.

**Review by HaraldPanten — APPROVED** [review]
Functionally is working fine :+1:

**Comment by ValentinVinagre** [comment]
Hi @rousseldenis,
The commit history is correct?

**Comment by rousseldenis** [comment]
@ValentinVinagre sounds good. But shouldn't two migration commits squashed into one?

File: stock_orderpoint_automatic_creation/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/153/11.0
```
Comment (sbidoul) [inline stock_orderpoint_automatic_creation/readme/USAGE.rst:3]: Do not put this image in USAGE.rst, it's now generated automatically as a readme badge.

File: stock_orderpoint_automatic_creation/models/product_template.py [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
 
     create_orderpoint = fields.Selection(
         selection=[
-            ('yes', 'Yes'),
-            ('no', 'No'),
+            ('yes', _('Yes')),
+            ('no', _('No')),
         ],
-        string='Create Orderpoint',
+        string=_('Create Orderpoint'),
```
Comment (sbidoul) [inline stock_orderpoint_automatic_creation/models/product_template.py:16]: Do not add these `_`, it's done automatically by Odoo AFAIR

**Review by sbidoul — CHANGES_REQUESTED** [review]
A few comments.

Also, it looks like your commits change the file permissions from 644 to 755 making them all executable. Can you fix that?

**Comment by rousseldenis** [comment]
@ValentinVinagre

**Comment by rousseldenis** [comment]
@ValentinVinagre Could you attend @sbidoul comments (it seems they are not resolved), check travis error and squash your commits ?

**Comment by rousseldenis** [comment]
@ValentinVinagre

**Pull request closed without merging**